transport/handler.go

```diff
@@ -22,7 +22,7 @@ package transport
 
 import "golang.org/x/net/context"
 
-// Handler handles a single transport-level request.
+// Handler handles a single, transport-level, unary request.
 type Handler interface {
```

Comment:
The more I see this the more I think the transport layer *should* actually refer to this as `UnaryHandler`.